code/__DEFINES/anomalies.dm

```diff
@@ -0,0 +1,18 @@
+// Anomaly core types
+/// Bluespace cores
+#define ANOMALY_CORE_BLUESPACE			/obj/item/assembly/signaler/anomaly/bluespace
+/// Gravitational cores
+#define ANOMALY_CORE_GRAVITATIONAL		/obj/item/assembly/signaler/anomaly/grav
+/// Flux
+#define ANOMALY_CORE_FLUX				/obj/item/assembly/signaler/anomaly/flux
+/// Vortex
+#define ANOMALY_CORE_VORTEX				/obj/item/assembly/signaler/anomaly/vortex
+/// Pyro
+#define ANOMALY_CORE_PYRO				/obj/item/assembly/signaler/anomaly/pyro
+
+// Max amounts of cores you can make
+#define MAX_CORES_BLUESPACE				8
+#define MAX_CORES_GRAVITATIONAL			8
+#define MAX_CORES_FLUX					8
+#define MAX_CORES_VORTEX				8
+#define MAX_CORES_PYRO					8
```

Comment:
The fact that not all of these line up makes me sad, but that might just be github